scripts/SimplexUI/loadUiType.py

```diff
@@ -21,39 +21,23 @@
 import xml.etree.ElementTree as xml
 from cStringIO import StringIO
 
-try:
-	from PySide import QtGui, QtCore
-	from PySide.QtCore import Signal, Slot
-	from PySide.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QSettings, QModelIndex
-	from PySide.QtGui import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QSortFilterProxyModel, QAction
-	from PySide.QtGui import QDialog, QMainWindow, QSplashScreen, QShortcut, QItemSelection, QProgressDialog
-	from PySide.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
-	import pysideuic as uic
-	def toPyObject(thing):
+def toPyObject(thing):
+	try:
+		return thing.toPyObject()
+	except:
 		return thing
 
+from Qt import QtGui, QtCore, QtWidgets, QtCompat
+from Qt.QtCore import Signal, QSortFilterProxyModel, Slot, QModelIndex
+from Qt.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QItemSelection, QSettings
+from Qt.QtWidgets import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QAction
+from Qt.QtWidgets import QDialog, QMainWindow, QSplashScreen, QShortcut, QProgressDialog
+from Qt.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
+try:
+	import pyside2uic as uic
 except ImportError:
-	try:
-		from PyQt4 import QtGui, QtCore
-		from PyQt4.QtCore import pyqtSignal as Signal, pyqtSlot as Slot
-		from PyQt4.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QSettings, QModelIndex
-		from PyQt4.QtGui import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QSortFilterProxyModel, QAction
-		from PyQt4.QtGui import QDialog, QMainWindow, QSplashScreen, QShortcut, QItemSelection, QProgressDialog
-		from PyQt4.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
-		import PyQt4.uic as uic
-		def toPyObject(thing):
-			return thing.toPyObject()
-
-	except ImportError:
-		from PySide2 import QtGui, QtCore
-		from PySide2.QtCore import Signal, QSortFilterProxyModel, Slot, QModelIndex
-		from PySide2.QtCore import Qt, QObject, QTimer, QPoint, QEvent, QItemSelection, QSettings
-		from PySide2.QtWidgets import QMessageBox, QInputDialog, QFileDialog, QMenu, QApplication, QAction
-		from PySide2.QtWidgets import QDialog, QMainWindow, QSplashScreen, QShortcut, QProgressDialog
-		from PySide2.QtGui import QStandardItemModel, QStandardItem, QKeySequence, QCursor, QMouseEvent
-		import pyside2uic as uic
-		def toPyObject(thing):
-			return thing
+	uic = QtCompat.loadUi
```

Comment:
Here's this in case it helps clarify.

```
ui_file = utils.get_ui_file(__file__)
MainWindow = QtCompat.loadUi(ui_file)
class SimplexDialog(MainWindow):
	def __init__(self, parent=None, dispatch=None):
```

```
# Error: TypeError: file C:\\dev\\Simplex\\scripts\SimplexUI\simplexdialog.py line 231: Error when calling the metaclass bases
    QMainWindow() takes at most 2 arguments (3 given) # 
```

It's late so whatever you want, I can work on implementing tomorrow.

But I got all the instances of Qt.\* imports separated into the modules and most of the dependency on loadUiType taken out, with the exception of this.